### ISSUE TITLE
Cannot reproduce the results in the paper

### ISSUE THREAD

**Issue by eslambakr** [sega-hsj/MVT-3DVG#10]
Hi everyone,

I run the training code with command provided in README, the results are lower than that in the paper, as shown in the below table:

Accuracy | MVT | Reproduce
-- | -- | --
Nr3d | 55.1 | 51.52
Sr3D | 64.5 | 62.3

Therefore, I am wondering if someone else encounter such issue while reproducing the results.

Thanks in advcne!

**Comment by sega-hsj** [comment]
Here we provide our checkpoints for Nr3d and Sr3d, which can reproduce the paper results.

https://drive.google.com/file/d/1ik-4Pbg2c69ufOcKl_bCHKmIbs8dL1ZD/view?usp=sharing
https://drive.google.com/file/d/1x4tCeHcdyo4p-JFnVnlHNlemPF6sWGWV/view?usp=sharing

The checkpoints include detailed configs and logs, you can take them as a reference to ensure you run the experiments correctly.
